agents/agent.py

```diff
@@ -20,6 +21,55 @@
     'agents.utils.logtools', 'logconfig.ini'
 )
 
+MAX_RETRIES = 3
+RETRY_DELAY = 5
+
+
+def log_error(msg):
+    with open('agent_log.txt', 'a') as f:
+        timestamp = time.strftime('%Y-%m-%d %H:%M:%S')
+        f.write('[ERROR] %s - %s\n' % (timestamp, msg))
```

Comment:
First and foremost: in the current implementation of agents, they come with their own loggers - why not use them?

I see how the task is formulated, but it was described before agent infrastructure was implemented, so we should be smart about it: there is no reason why periodic post errors can't be logged in the existing log files (using `maybe_log_message`).